packages/experimental/Expander/src/Expander.tsx

```diff
@@ -0,0 +1,50 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { expanderName, ExpanderType, ExpanderProps, ExpanderViewProps } from './Expander.types';
+import { compose, mergeProps, withSlots, UseSlots, buildProps } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+
+const ExpanderComponent = ensureNativeComponent('MSFExpanderView');
+function delay(ms: number) {
+  return new Promise( resolve => setTimeout(resolve, ms) );
+}
+
+export const Expander = compose<ExpanderType>({
+  displayName: expanderName,
+  tokens: [{}, expanderName],
+  slotProps: {
+    root: buildProps(
+      (tokens) => ({...tokens})
+    ),
+  },
+  slots: { root: ExpanderComponent },
+  render: (userProps: ExpanderProps, useSlots: UseSlots<ExpanderType>) => {
+    const Root = useSlots(userProps).root;
+    const [expandedState, setExpandedState] = React.useState(userProps.expanded);
+    const expanderHeight = expandedState? userProps.expandedHeight : userProps.collapsedHeight;
+
+    const _onCollapsing = async () => {
+      if (userProps.onCollapsing !== undefined) {
+        userProps.onCollapsing();
+      }
+      // Need to delay the height change so that the animation runs
```

Comment:
is this a pattern rn-windows folks have been taking to generate layout animations w/o using LayoutAnimation [due to limitations e.g. its global nature] ? This is a 175ms wait on the JS thread, no?

Reply:
Yes, we added a wait on the JS thread to allow enough time for the native animation to complete before changing the height of the Expander.

Reply:
@ppatboyd RNW does not support RN layout animations. All animations here are XAML animations.